src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -922,7 +922,7 @@ export default {
           title: this.filingTypeToName(filingType),
           subtitle,
           filingId: header.filingId,
-          filingAuthor: 'Registry Staff', // TBD
+          filingAuthor: 'Registry Staff',
```

Comment:
As far as I know, alteration filings are filed by Registry Staff (Ops) only.

Reply:
Wait, is this true? 
Doesn't the client themselves file the Alterations? The staff only load the business and create the draft right?

Reply:
I'll review this tomorrow!

Reply:
You're right: alterations can be filed by users. And that code is correct (on line 684)...

Line 925 (above) is actually for the Transition Filings. I typed in the wrong filing type in my comment :P

Reply:
Sounds good! You can resolve when ready 👍